src/models/connection/from_client/connection_from_client.ts

```diff
@@ -32,22 +37,43 @@ export class ConnectionFromClient {
     } else if (obj.secondTool.tool == ToolType.REDMINE.name) {
       this.checkRedmineUrl(obj.secondTool);
     }
+
+    if (obj.firstTool.tool == ToolType.JIRA.name) {
+      this.checkJiraUrl(obj.firstTool);
+    } else if (obj.secondTool.tool == ToolType.JIRA.name) {
+      this.checkJiraUrl(obj.secondTool);
+    }
+
+
     this.firstTool = obj.firstTool;
     this.secondTool = obj.secondTool;
+    this.projectMappings = obj.projectMappings
   }
 
   private checkRedmineUrl(tool: any) {
     tool.redmineApiPoint = tool.redmineApiPoint.endsWith('/')
       ? tool.redmineApiPoint
       : `${tool.redmineApiPoint}/`;
+
+    tool.redmineApiPoint = tool.redmineApiPoint.startsWith('https://')
+      ? tool.redmineApiPoint
+      : `https://${tool.redmineApiPoint}`
```

Comment:
Tohle se mi úplně nelíbí. Docela to ztěžuje lokální testování kde mám většinou RM na něčem jako localhost:9000 :) Pokud to ale nejde udělat jinak tak se s tím smířím.

Reply:
řešeno o komment vedle